### DISCUSSION TITLE
Minimum OS Support

### DISCUSSION THREAD

**Discussion by slannon97** [samtupy/nvgt#130] [General]
The subject says it all. What are the minimum supported versions of operating systems that NVGT applications can run on?

**Comment by harrymkt** [comment]
I believe NVGT and/or its applications could run on:
* All Windows versions, provided that they are 64 bit. If not all, Windows 7 or later.
* Android 10 or later (64 bit). I think the engine should have the ability to set minimum required Android version but it is not available yet.

For other 2 supported systems, I don't know, because I have not tested them personally.

**Reply by aryanchoudharypro** [reply]
hi, just here to noat that nvgt can run on android 5 and lator

**Reply by harrymkt** [reply]
Good to know then. Thanks.
